### PR TITLE
added content template, and default

### PR DESCRIPTION
**Pull request by AdithyaPNCodilar**

### CODE REVIEW

File: templates/content/product.html.twig [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    <div class="product-info">
+        {% if editmode %}
+            {{ pimcore_relation("product") }}
+        {% else %}
+            {% set product = pimcore_relation("product").element %}
+            {% if product %}
+                <h2>{{ product.name }}</h2>
+                <div class="content">
+                    {% if product.image %}
+                        {{ product.image.thumbnail("kitkat").html|raw }}
```
Comment (abhishek-omnia) [inline templates/content/product.html.twig:14]: what is this kitkat

File: templates/layout.html.twig [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+            width: 600px;
+            padding: 30px 0 0 0;
+            color: #65615E;
+        }
+
+        h1, h2, h3 {
+            font-size: 18px;
+            padding: 0 0 5px 0;
+            border-bottom: 1px solid #001428;
+            margin-bottom: 5px;
```
Comment (abhishek-omnia) [inline templates/layout.html.twig:25]: keep css in different file

File: templates/content/product.html.twig (hunk 1 above)
Comment (abhiramalli) [inline templates/content/product.html.twig]: what is kitkat

File: templates/layout.html.twig [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+    <meta charset="UTF-8">
+    <title>Example</title>
+    <style>
+        body {
+            padding: 0;
+            margin: 0;
+            font-family: "Lucida Sans Unicode", Arial;
+            font-size: 14px;
+        }
+
```
Comment (abhiramalli) [inline templates/layout.html.twig]: keep css in different file

**Pull request merged**